package.json

```diff
@@ -53,60 +53,60 @@
     }
   },
   "peerDependencies": {
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0",
-    "styled-components": "^5.0.0"
+    "react": "^16.14.0",
+    "react-dom": "^16.14.0",
+    "styled-components": "^5.3.6"
   },
   "devDependencies": {
-    "@babel/core": "^7.7.2",
-    "@babel/plugin-transform-runtime": "^7.8.3",
-    "@storybook/addon-options": "^5.3.4",
-    "@storybook/addons": "^5.3.4",
-    "@storybook/react": "^5.3.4",
-    "@storybook/theming": "^5.3.4",
-    "@types/jest": "^24.0.22",
-    "@types/lodash.get": "^4.4.6",
-    "@types/lodash.isequal": "^4.5.5",
-    "@types/react": "^16.9.11",
-    "@types/react-redux": "^7.1.5",
-    "@types/react-test-renderer": "^16.9.1",
-    "@types/simonwep__selection-js": "^1.2.0",
-    "@types/storybook-readme": "5.0.4",
-    "@types/styled-components": "5.0.1",
-    "ace-builds": "^1.4.7",
+    "@babel/core": "^7.20.2",
+    "@babel/plugin-transform-runtime": "^7.19.6",
+    "@storybook/addon-options": "^5.3.21",
+    "@storybook/addons": "^5.3.22",
+    "@storybook/react": "^5.3.21",
+    "@storybook/theming": "^6.5.14",
+    "@types/jest": "^29.2.4",
+    "@types/lodash.get": "^4.4.7",
+    "@types/lodash.isequal": "^4.5.6",
+    "@types/react": "^16.14.34",
+    "@types/react-redux": "^7.1.24",
+    "@types/react-test-renderer": "^18.0.0",
+    "@types/simonwep__selection-js": "^1.7.0",
+    "@types/storybook-readme": "^5.0.6",
+    "@types/styled-components": "^5.1.26",
+    "ace-builds": "^1.13.2",
     "awesome-typescript-loader": "^5.2.1",
-    "babel-loader": "^8.0.6",
-    "browserify": "^16.5.0",
-    "commitizen": "^4.0.4",
-    "cz-emoji": "^1.1.2",
-    "gh-pages": "^2.2.0",
-    "jest": "^25.1.0",
+    "babel-loader": "^8.3.0",
+    "browserify": "^17.0.0",
+    "commitizen": "^4.2.6",
+    "cz-emoji": "^1.3.1",
+    "gh-pages": "^4.0.0",
+    "jest": "^25.5.4",
     "lodash.noop": "^3.0.1",
     "markdown-loader": "^5.1.0",
```

Comment:
(Use `node --trace-deprecation ...` to show where the warning was created)
/mnt/fdf8f571-43ba-4785-b8dd-16c63e8595f9/projects/ramen/node_modules/html-webpack-plugin/lib/webpack5/file-watcher-api.js:13
    mainCompilation.fileSystemInfo.createSnapshot(
                                   ^

TypeError: Cannot read properties of undefined (reading 'createSnapshot')
    at /mnt/fdf8f571-43ba-4785-b8dd-16c63e8595f9/projects/ramen/node_modules/html-webpack-plugin/lib/webpack5/file-watcher-api.js:13:36
    at new Promise (<anonymous>)
    at Object.createSnapshot (/mnt/fdf8f571-43ba-4785-b8dd-16c63e8595f9/projects/ramen/node_modules/html-webpack-plugin/lib/webpack5/file-watcher-api.js:12:10)
    at /mnt/fdf8f571-43ba-4785-b8dd-16c63e8595f9/projects/ramen/node_modules/html-webpack-plugin/lib/cached-child-compiler.js:219:35